src/session/content/chat_action_bar.rs

```diff
@@ -367,12 +368,19 @@ impl ChatActionBar {
 
     fn load_message_to_edit(&self, message_id: i64) {
         if let Some(chat) = self.chat() {
+            let client_id = chat.session().client_id();
+
             if let Some(message) = chat.history().message_by_id(message_id) {
                 match message.content().0 {
                     MessageContent::MessageText(data) => {
-                        self.imp()
-                            .message_entry
-                            .set_formatted_text(Some(BoxedFormattedText(data.text)));
+                        block_on(async move {
+                            let FormattedText::FormattedText(markdown_text) =
+                                get_markdown_text(data.text, client_id).await.unwrap();
```

Comment:
Why not just `functions::get_markdown_text`?

Reply:
I actually don't see a reason for exposing the entire functions namespace instead of just the specific function.

I would do that if there's an ambiguity issue, like using `FormattedText` from `tdlib::enums` and `tdlib::types` at the same time.

Reply:
Alright, actually, `tdlib::functions` was already imported, so it makes sense. Thanks.